static/app/components/platformPicker.tsx

```diff
@@ -108,7 +111,11 @@ class PlatformPicker extends React.Component<Props, State> {
               <ListLink
                 key={id}
                 onClick={(e: React.MouseEvent) => {
-                  analytics('platformpicker.select_tab', {category: id});
+                  trackAdvancedAnalyticsEvent(
+                    'growth.onboarding_platform_category',
+                    {category: id},
+                    this.props.organization ?? null
```

Comment:
@roggenkemper I'm seeing `organization` coming out as `undefined` in the analytics event here. I think we aren't passing it down the component tree.